cmake/Modules/LocalJpeg.cmake

```diff
@@ -1,7 +1,18 @@
-add_subdirectory(${AVIF_SOURCE_DIR}/ext/libjpeg EXCLUDE_FROM_ALL)
+set(LIB_DIR "${AVIF_SOURCE_DIR}/ext/libjpeg-turbo/build.libavif")
+if(WIN32)
```

Comment:
Can we make this MSVC specific please? MinGW toolcjains have no problem distinguishing static .a from stufb .dll.a files and don't need the "static" suffix workaaround.

Reply:
Did our MinGW CI builds fail when we were checking `if(WIN32)` here?

Reply:
Thank you.